src/test/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionServiceTests.java

```diff
@@ -433,13 +431,13 @@ void shouldGetPostComputePalaemonTokens() throws Exception {
                 .thenReturn(POST_COMPUTE_ENTRYPOINT);
         when(web2SecretsService.getSecret(
                 request.getTaskDescription().getBeneficiary(),
-                ReservedSecretKeyName.IEXEC_RESULT_ENCRYPTION_PUBLIC_KEY,
+                IEXEC_RESULT_ENCRYPTION_PUBLIC_KEY,
```

Comment:
I find it more readable as it was. Especially since we import a lot of static elements from different classes.

Reply:
Fixed in ec97612

Reply:
Thanks!